gm4_live_catch/data/gm4_live_catch/functions/fish/tropical_fish/summon.mcfunction

```diff
@@ -0,0 +1,15 @@
+# run from gm4_live_catch:fish/FISH_TYPE/target
+
+
+# store nbt
+data modify storage gm4_live_catch:temp target set from entity @s
```

Comment:
This will allow you to merge the entire storage into the fish after adding the Air and Health modifications to the storage.

```suggestion
data modify storage gm4_live_catch:temp target.Motion set from entity @s Motion
```